Topics/Topic_2_b/src/main/java/com/bootcamp/topic2_b/blog/Blog.java

```diff
@@ -0,0 +1,87 @@
+package com.bootcamp.topic2_b.blog;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Blog {
+	private List<Entry> postsList;
+	
+	public Blog(ArrayList<Entry> postsList){
+		this.postsList = postsList;
+	}
+	
+	public Blog(){
+		postsList = new ArrayList<Entry>();
+	}
+	
+	public Entry getEntry(int idEntry){
+		for (Entry entry : postsList) {
+			if(entry.getId()==idEntry)
+				return entry;
+		}
+		return null;
+	}
+	
+	public void removeEntry(int idEntry){
+		for (Entry entry : postsList) {
+			if(entry.getId()==idEntry){
+				postsList.remove(entry);
+				break;
+			}
+		}
+	}
+	
+	public void addEntry(Entry entry){
+		postsList.add(entry);
+		this.sortPostsList();
+	}
+	
+	public void updateEntry(Entry entry){ 			
+		boolean needToBeSorted = false;				//This works only if the Entry object
+		for (Entry iEntry : postsList) {			// of the parameter has the same id
+			if(entry.getId()==iEntry.getId()){		// of the object in postsList.
+				needToBeSorted = !(entry.getDate().equals(iEntry.getDate()));
+				iEntry = entry;
+			}
+		}
+		if(needToBeSorted) 
+			this.sortPostsList();
+	}
+	
+	public Blog getRecentPosts(int numberOfPosts){

```

Comment:
At this commit, this method returns a Blog object with a List object which has inside the oldest posts!
If you see, I've got a new commit where I fix this problem, and now, it returns the requested Blog with its corresponding List object.